src/bindings/python/flux/cli/base.py

```diff
@@ -388,6 +388,8 @@ class BatchConfig:
     Iteratively build a "config" dict from successive updates.
     """
```

Comment:
commit message "useful to users under /etc/xdg/flux/config/*.toml" should be `{*.toml, *.json}`?